exo-go/src/aws/terraform_helpers.go

```diff
@@ -0,0 +1,82 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/s3"
+)
+
+// InitAccount prepares a blank AWS account to be used with Terraform
+func InitAccount(bucketName, tableName, region string) error {
+	config := aws.NewConfig().WithRegion(region)
+	session := session.Must(session.NewSession())
+
+	err := createRemoteState(session, config, bucketName)
+	if err != nil {
+		return err
+	}
+
+	return createLockTable(session, config, tableName)
+}
+
+// creates s3 bucket to store terraform remote state
+func createRemoteState(currSession *session.Session, config *aws.Config, bucketName string) error {
+	s3client := s3.New(currSession, config)
+
+	hasBucket, err := hasBucket(s3client, bucketName)
+	if err != nil {
+		return err
+	}
+
+	if !hasBucket {
+		_, err = s3client.CreateBucket(&s3.CreateBucketInput{Bucket: &bucketName})
+		if err != nil {
+			return err
+		}
+
+		// enable versioning
+		versioningStatus := "Enabled"
```

Comment:
I saw you using `aws.String` elsewhere can that be used here instead of declaring the variable?